src/ApplySecureHeaders.php

```diff
@@ -132,4 +125,35 @@ private function setExpectCT()
             );
         }
     }
+
+    /**
+     * Set any protected cookies.
+     */
+    private function setProtectedCookies()
+    {
+        $this->headers->protectedCookie('lol');
+        // $namesToAdd = $this->config->get('secure-headers.protectedCookies.add.names', []);
+
+        // if (count($namesToAdd)) {
+        // $this->headers->protectedCookie($namesToAdd);
```

Comment:
For exact names, make sure to pass the [`SecureHeaders::COOKIE_NAME`](https://github.com/aidantwoods/SecureHeaders/wiki/protectedCookie#cookie_name) constant in the second arg (and similar when removing names, but combined (bitwise OR) with `SecureHeaders::COOKIE_REMOVE`).

I think when I get around to v3 I'll remove all this bitwise stuff, not terribly straightforward compared to e.g. `addProtectedCookieSubstring`, `removeProtectedCookieSubstring` :)

Reply:
Will do, this is very much WIP - just wanted to get a bit of help on the set cookie "issue"